Documentation/UsingSetting/PhpApi.rst

```diff
@@ -1,74 +1,83 @@
-.. include:: /Includes.rst.txt
-.. index::
-   Page TSconfig; PHP API
-   Page TSconfig; Retrieving
-   Page TSconfig; Getting
+..  include:: /Includes.rst.txt
+..  index::
+    Page TSconfig; PHP API
+    Page TSconfig; Retrieving
+    Page TSconfig; Getting
 
-.. _phpapi:
+..  _phpapi:
 
 =======
 PHP API
 =======
 
-The PHP API to retrieve page and user TSconfig in a backend module is straight
-forward:
+Retrieving TSconfig settings
+============================
 
-.. code-block:: php
-   :caption: EXT:my_sitepackage/Classes/Controller/MyBackendController.php
+The PHP API to retrieve page and user TSconfig in a backend module is
+straightforward:
```

Comment:
```suggestion
straightforward:
```
Is one of the words that we wanted to avoid as it makes ppl who dont find this easy feel bad?